onnx/defs/controlflow/defs.cc

```diff
@@ -590,7 +590,8 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Output(
             0,
             "v_final_and_scan_outputs",
-            "Final N loop carried dependency values then K scan_outputs",
+            "Final N loop carried dependency values then K scan_outputs. "
+            "Scan outputs must be Tensors.",
```

Comment:
I think the inference function above does not check that scan-outputs are tensors. Can we make the check stricter for scan outputs?

Reply:
updated